lighthouse-core/test/audits/seo/http-status-code-test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../../../audits/seo/http-status-code.js');
```

Comment:
nit: rename as `httpStatusCodeAudit`? It gets confusing when reading the tests if just generic `Audit`. (I regret we haven't switched all the rest of the audit tests off of using `Audit` for this)

Reply:
👍